<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DogShelterQTClass</class>
 <widget class="QMainWindow" name="DogShelterQTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DogShelterQT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <layout class="QVBoxLayout" name="leftLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QComboBox" name="filterCombo">
          <item>
           <property name="text">
            <string>Detailed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Short</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="dogListWidget"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonLayout">
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="updateButton">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="breedLabel">
            <property name="text">
             <string>Breed:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="breedLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="nameLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="ageLabel">
            <property name="text">
             <string>Age:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="ageLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="linkLabel">
            <property name="text">
             <string>Link:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="linkLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="toUserMode">
          <property name="text">
           <string>Go to User Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="undoButton">
            <property name="text">
             <string>Undo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="redoButton">
            <property name="text">
             <string>Redo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="statButton">
          <property name="text">
           <string>Statistics</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QLabel" name="nextLabel">
          <property name="text">
           <string>Current Dog:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="nextLayout">
          <item>
           <widget class="QLineEdit" name="nextDogLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="nextButton">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="adoptCurrent">
            <property name="text">
             <string>Adopt this dog</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="adoptedListWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="filterLabel">
          <property name="text">
           <string>Filter details:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="filterForm">
          <item row="1" column="0">
           <widget class="QLabel" name="breedLabel2">
            <property name="text">
             <string>Breed:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="breedLineEditFilter">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="ageLabel2">
            <property name="text">
             <string>Age:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ageLineEditFilter"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="filterButton">
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fileNameLabel">
          <property name="text">
           <string>File name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileNameLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="savetoFileButton">
          <property name="text">
           <string>Save to file</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="adoptAll">
            <property name="text">
             <string>Adopt all dogs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="toAdminMode">
            <property name="text">
             <string>Go to Admin Mode</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuStatistics">
    <property name="title">
     <string>Statistics</string>
    </property>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuStatistics"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dogshelterqt.qrc"/>
 </resources>
 <connections/>
</ui>
